google/cloud/storage/client.h

```diff
@@ -59,24 +59,43 @@ inline namespace STORAGE_CLIENT_NS {
  * this class is not guaranteed to work.
  *
  * @par Credentials
- * The default constructor for this class will attempt to load Application
- * Default %Credentials (ADCs). If you wish to use no credentials or to supply a
- * specific Credentials type, you can use the functions declared in
- * google_credentials.h:
+ * The default approach for creating a Client uses Google Application Default
+ * %Credentials. Because finding or loading ADCs can fail, the returned
```

Comment:
`ADCs` is used before being defined, please consider restoring the definition in parenthesis.

Reply:
Added parenthetical back; fixed.